components/org.wso2.carbon.identity.api.server.workflow.management/org.wso2.carbon.identity.api.server.workflow.management.v1/src/main/java/org/wso2/carbon/identity/workflow/mgt/api/v1/util/WorkflowManagementEndpointUtil.java

```diff
@@ -0,0 +1,87 @@
+//package org.wso2.carbon.identity.workflow.mgt.api.v1.util;
+//
+//import org.apache.commons.httpclient.URI;
+//import org.apache.commons.logging.Log;
+//import org.apache.commons.logging.LogFactory;
+//import org.wso2.carbon.identity.workflow.mgt.api.v1.exceptions.WorkflowManagementEndpointException;
+//
+//import javax.ws.rs.core.Response;
+//
+//public class WorkflowManagementEndpointUtil {
```

Comment:
remove this class if not needed

Reply:
fixed by : https://github.com/wso2/identity-api-server/pull/446/commits/3f0c88651577bd8b27c0f5d34ba6922881bcfe38